app/services/create_location_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLocationDecorator
```

Comment:
Having a service object to create a decorated object doesn't quite feel right.

I'm wondering whether the service could return a value object containing `location`, `booking_location`, `search_context` and `twilio_number` then the controller can take the responsibility of decorating for the view it is trying to render?